app/models/piece.rb

```diff
@@ -18,12 +17,47 @@ def move!(x, y)
     true
   end
 
+  # Return false if coordinates is taken but belongs to friendly piece
+  # otherwise return true
+  def capturable?(x, y)
+    return false unless position_taken?(x, y) && !friendly_piece?(x, y)
+    true
+  end
+
+  # if piece is capturable change piece's attributes
+  def capture!(x, y)
+    return false unless capturable?(x, y)
+    move!(x, y)
+    target_piece(x, y).update_attributes!(
+      x_position: nil,
+      y_position: nil,
+      captured: true
+    )
+  end
```

Comment:
Since the meeting is tomorrow, let's discuss it then when new tickets are created.  In the meantime, the are plenty of tickets that you can and should grab from the backlog to work on that, in my opinion, are more urgent.